src/SFML/Window/Win32/InputImpl.cpp

```diff
@@ -48,113 +48,149 @@ bool InputImpl::isKeyPressed(Keyboard::Key key)
-        case Keyboard::Equal:      vkey = VK_OEM_PLUS;   break;
-        case Keyboard::Dash:       vkey = VK_OEM_MINUS;  break;
-        case Keyboard::Space:      vkey = VK_SPACE;      break;
-        case Keyboard::Return:     vkey = VK_RETURN;     break;
-        case Keyboard::BackSpace:  vkey = VK_BACK;       break;
-        case Keyboard::Tab:        vkey = VK_TAB;        break;
-        case Keyboard::PageUp:     vkey = VK_PRIOR;      break;
-        case Keyboard::PageDown:   vkey = VK_NEXT;       break;
-        case Keyboard::End:        vkey = VK_END;        break;
-        case Keyboard::Home:       vkey = VK_HOME;       break;
-        case Keyboard::Insert:     vkey = VK_INSERT;     break;
-        case Keyboard::Delete:     vkey = VK_DELETE;     break;
-        case Keyboard::Add:        vkey = VK_ADD;        break;
-        case Keyboard::Subtract:   vkey = VK_SUBTRACT;   break;
-        case Keyboard::Multiply:   vkey = VK_MULTIPLY;   break;
-        case Keyboard::Divide:     vkey = VK_DIVIDE;     break;
-        case Keyboard::Left:       vkey = VK_LEFT;       break;
-        case Keyboard::Right:      vkey = VK_RIGHT;      break;
-        case Keyboard::Up:         vkey = VK_UP;         break;
-        case Keyboard::Down:       vkey = VK_DOWN;       break;
-        case Keyboard::Numpad0:    vkey = VK_NUMPAD0;    break;
-        case Keyboard::Numpad1:    vkey = VK_NUMPAD1;    break;
-        case Keyboard::Numpad2:    vkey = VK_NUMPAD2;    break;
-        case Keyboard::Numpad3:    vkey = VK_NUMPAD3;    break;
-        case Keyboard::Numpad4:    vkey = VK_NUMPAD4;    break;
-        case Keyboard::Numpad5:    vkey = VK_NUMPAD5;    break;
-        case Keyboard::Numpad6:    vkey = VK_NUMPAD6;    break;
-        case Keyboard::Numpad7:    vkey = VK_NUMPAD7;    break;
-        case Keyboard::Numpad8:    vkey = VK_NUMPAD8;    break;
-        case Keyboard::Numpad9:    vkey = VK_NUMPAD9;    break;
-        case Keyboard::F1:         vkey = VK_F1;         break;
-        case Keyboard::F2:         vkey = VK_F2;         break;
-        case Keyboard::F3:         vkey = VK_F3;         break;
-        case Keyboard::F4:         vkey = VK_F4;         break;
-        case Keyboard::F5:         vkey = VK_F5;         break;
-        case Keyboard::F6:         vkey = VK_F6;         break;
-        case Keyboard::F7:         vkey = VK_F7;         break;
-        case Keyboard::F8:         vkey = VK_F8;         break;
-        case Keyboard::F9:         vkey = VK_F9;         break;
-        case Keyboard::F10:        vkey = VK_F10;        break;
-        case Keyboard::F11:        vkey = VK_F11;        break;
-        case Keyboard::F12:        vkey = VK_F12;        break;
-        case Keyboard::F13:        vkey = VK_F13;        break;
-        case Keyboard::F14:        vkey = VK_F14;        break;
-        case Keyboard::F15:        vkey = VK_F15;        break;
-        case Keyboard::Pause:      vkey = VK_PAUSE;      break;
```

Comment:
Looks like your IDE auto "corrected" the indention here, unfortunately it's not how we format switch-cases.

Reply:
Should be fixed now